src/api/messages/getMessages.js

```diff
@@ -74,19 +74,22 @@ const migrateResponse = response => {
  */
 export default async (
   auth: Auth,
-  narrow: Narrow,
-  anchor: number,
-  numBefore: number,
-  numAfter: number,
-  useFirstUnread: boolean = false,
+  args: {
+    narrow: Narrow,
+    anchor: number,
+    numBefore: number,
+    numAfter: number,
+    useFirstUnread?: boolean,
+  },
 ): Promise<ApiResponseMessages> => {
+  const { narrow, anchor, numBefore, numAfter, useFirstUnread } = args;
   const response: ServerApiResponseMessages = await apiGet(auth, 'messages', {
     narrow: JSON.stringify(narrow),
     anchor,
     num_before: numBefore,
     num_after: numAfter,
     apply_markdown: true,
-    use_first_unread_anchor: useFirstUnread,
+    use_first_unread_anchor: useFirstUnread || false,
```

Comment:
You can actually still set a default value for useFirstUnread, in the destructuring assignment, like so, so you don't have to modify code wherever it's consumed. In this case, that's just one place, but this would be useful if a lot of things used it. 🙂 

```
const { narrow, anchor, numBefore, numAfter, useFirstUnread = false } = args;
```

Here's a doc describing this syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Setting_a_function_parameters_default_value

Reply:
Done! Thanks for the doc link!